benefits/core/migrations/0002_data.py

```diff
@@ -220,6 +232,23 @@ def load_data(app, *args, **kwargs):
         start_template="eligibility/start--senior.html",
     )
 
+    sbmtd_mobility_pass_verifier = EligibilityVerifier.objects.create(
+        name=os.environ.get("MOBILITY_PASS_VERIFIER_NAME", "Eligibility Server Verifier"),
+        active=os.environ.get("MOBILITY_PASS_VERIFIER_ACTIVE", "True").lower() == "true",
```

Comment:
Sorry just noticed something else.

We need an app config and key vault secrets for this `active` setting as well.

And let's go ahead and make the default `False` here so it doesn't turn on by accident.

Reply:
@machikoyasuda we'll need a follow-up PR for this.

Reply:
Actually, all the `MOBILITY_PASS_*` environment variables need key vault secrets defined in Terraform. I'll create a new issue.